src/pools/pool.service.ts

```diff
@@ -197,4 +199,24 @@ export class PoolService {
 
     return (this.pool.isNew = isNew);
   }
+
+  /**
+   * @summary Calculates absolute max APR given boost or not.
+   * If given boost returns user's max APR.
+   * If not given boost returns pool absolute max assuming 2.5x boost.
+   * Used primarily for sorting tables by the APR column.
+   */
+  absMaxApr(aprs: AprBreakdown, boost?: string): string {
```

Comment:
This code should be in the SDK rather than here. The boost could be a secondary argument (maybe an options argument) to `this.pools.apr`. Can you talk to @gmbronco about adding it. 

What do you mean by "users max APR" as this is a generic calculation, it can't be made user specific.